safe-core-sdk/auth-kit/guides/safe-auth.md

```diff
@@ -0,0 +1,179 @@
+# Integration with SafeAuth
+
+This guide demonstrates creating an externally-owned account using your email or social media account. Once authenticated, you can sign transactions and interact with your Safe accounts.
+
+The `SafeAuthPack` is an authentication system that utilizes the [Web3Auth](https://web3auth.io) MPC technology. It was developed in collaboration with Web3Auth to create a smooth onboarding experience for web2 users across different dApps.
```

Comment:
Could you add onboarding to Vale's allowlist?

Reply:
done